index.js

```diff
@@ -60,6 +60,7 @@ function Slideout(options) {
   this._opened = false;
   this._preventOpen = false;
   this._touch = options.touch === undefined ? true : options.touch && true;
+  this._touchEnabled = true;
```

Comment:
fair point, I can make that change. i guess the better solution would be to add and remove the touch events, but that seemed like a bigger and more error prone change.